client/src/pages/StepForm/WhatYouWillNeedToKnow.js

```diff
@@ -0,0 +1,66 @@
+import {
+  Grid as G,
+  Typography as T,
+  Inputs as I,
+  TextWithIcon,
+} from '../../components';
+
+const newItem = { title: '', description: '', things: [''], tips: [''] };
+
+const WhatYouWillNeedToKnow = ({ state, setState, errors }) => {
+  return (
+    <>
+      <G.Row mt="8">
+        <G.Col w={[4, 4, 4]}>
+          <T.H2>Things you’ll need</T.H2>
```

Comment:
This should be What you'll need to know  (you've done things you'll need twice)

Reply:
done